shell/common/engine.cc

```diff
@@ -201,9 +216,16 @@ void Engine::RunBundle(const std::string& bundle_path) {
       runtime_->dart_controller()->RunFromKernel(kernel);
       return;
     }
+    std::string snapshot_path = DirPath(bundle_path) + "/snapshot_blob.bin";
```

Comment:
Use `files::GetDirectoryName()` instead of `DirPath()`? And maybe `blink::kSnapshotAssetKey` instead of `snapshot_blob.bin`?